spec/classes/collectd_plugin_disk_spec.rb

```diff
@@ -48,6 +48,25 @@
     end
   end
 
+  context ':manage_package => false on osfamily => RedHat' do
+    let :facts do
+      {
+        osfamily: 'RedHat',
+        collectd_version: '5.5',
+      }
+    end
+
+    let :params do
+      {
+        manage_package: false,
+      }
+    end
+    it 'Will not manage collectd-disk' do
+      should_not contain_package('collectd-disk').with(ensure: 'present',
```

Comment:
Thanks for the info. I copied it from here: https://github.com/voxpupuli/puppet-collectd/blob/master/spec/classes/collectd_plugin_disk_spec.rb#L73 :)